learning.yml

```diff
@@ -112,9 +111,9 @@
 
     - role: ome.omero_web_apps
       omero_web_apps_packages:
-        - omero-gallery
-        - omero-iviewer
-        - omero-virtual-microscope
+        - omero-gallery==3.1.1
+        - omero-iviewer==0.8.1
+        - omero-virtual-microscope==1.0.1
```

Comment:
Not for this PR
I don't think we look at ``omero-virtual-microscope`` in the context of py3
Have you checked @mtbc?

Reply:
This is from the extra commits needed to get the other PR approved that then conflicted with this one so I merged it in.

I made sure that `omero-virtual-microscope` on the gdoc spreadsheet of apps. I don't yet know what it needs before it will work.